app/views/entities/index.html.erb

```diff
@@ -0,0 +1,36 @@
+<nav class="bg-blue-500 py-4">
+  <div class="container mx-auto px-4 flex justify-between items-center">
+    <div>
+      <%= link_to(groups_path, class: "text-white px-2 py-2 rounded-lg hover:bg-blue-700") do %>
+        <i class="fas fa-arrow-left mr-2"></i><% end %>
+    </div>
+
+    <!-- Centered page heading -->
+    <div class="flex-1 text-center text-white text-2xl font-semibold">
+      Transactions
+    </div>
+  </div>
+</nav>
+
+<div class="container mx-auto px-4 mt-4">
+  <div class="text-center text-lg font-semibold">
+    Total Amount of Transaction: <%= @total_amount %> $
+  </div>
+</div>
+
```

Comment:
- I think it would be great if you make the order of the transaction by the most recent as mentioned in the requirements since when I add a new expense it comes after it but it should be above the past transaction so kindly fix this as well ✨💪🏻🔥


![image](https://github.com/Rana-Mudassir/rails-capstone-budget-app/assets/58553711/8470152e-fca4-43fe-a343-09777fe931a4)

- Also I added ships after burger but comes after but it should be the first expense to have as a recent 🔥
![image](https://github.com/Rana-Mudassir/rails-capstone-budget-app/assets/58553711/f5ae4d00-026d-4557-bab8-f047de978d52)

Reply:
Thanks for the feeback. It's working fine now, you can see the latest transaction on the top now.

Reply:
Fixed.